src/components/MDXComponents/EmailSubscriptionForm.tsx

```diff
@@ -0,0 +1,138 @@
+import Box from "@mui/material/Box"
+import TextField from "@mui/material/TextField"
+
+
+type FormType = {
+  email: string;
+  loading: boolean;
+  error: null | "invalidEmail" | "networkError";
+};
+
+
+// Collect email address that subscribe to TiDB release and send it to the SendGrid API
+function EmailSubscriptionForm() {
+  const { t, navigate } = useI18next();
+
+  const API_URL =
+    "https://03tryjsrrb.execute-api.us-west-2.amazonaws.com/Prod/subscribe/";
+
+  const [formData, setFormData] = useState<FormType>({
+    email: "",
+    loading: false,
+    error: null,
+  });
+  const [success, setSuccess] = useState(false);
+
+  const validateEmail = (): boolean => {
+    // Set up a schema for validating the email address
+    const schema = z.string().email();
+
+    try {
+      schema.parse(formData.email);
+      return true;
+    } catch (e) {
+      console.error(e);
+      setFormData({ ...formData, error: "invalidEmail" });
+      return false;
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+    setFormData((prevState) => ({ ...prevState, email: value }));
+  };
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (validateEmail()) {
+      // Submitting...
+      setFormData({ ...formData, loading: true, error: null });
+
+      try {
+
+        await axios.post(API_URL, undefined, {
+          email: formData.email
+        });
+
+        setFormData({ ...formData, loading: false, error: null });
+        setSuccess(true);
+      } catch (e) {
+        setFormData({ ...formData, loading: false, error: "networkError" });
+        console.error(e);
+      }
+    }
+  };
+
+  return (
+    <div>
+      <p>
+        <strong>
+          <Trans i18nKey="releaseSubscription.title" />:
+        </strong>
+      </p>
+
+      {!success ? (
+        <div>
+          <form onSubmit={handleSubmit}>
+            <Box display="flex" alignItems="center">
+              <TextField
+                size="small"
+                variant="outlined"
+                type="text"
+                name="email"
+                placeholder={t("releaseSubscription.placeholder")}
+                value={formData.email}
+                onChange={handleChange}
+                error={!!formData.error}
+                helperText={formData.error === "invalidEmail" ? t('releaseSubscription.error.invalidEmail') : t('releaseSubscription.error.networkError')}
+              />
+
+              <LoadingButton
+                type="submit"
+                variant="contained"
+                color="primary"
+                loading={formData.loading}
+                loadingIndicator={t('releaseSubscription.button.subscribing')}
+                ml="1rem"
+              >
+                <Trans i18nKey="releaseSubscription.button.subscribe" />
+              </LoadingButton>
+            </Box>
```

Comment:
I made a mistake. The correct codes are:

```suggestion
            <Box display="flex" alignItems="baseline">
              <TextField
                size="small"
                variant="outlined"
                type="text"
                name="email"
                placeholder={t("releaseSubscription.placeholder")}
                value={formData.email}
                onChange={handleChange}
                error={!!formData.error}
                helperText={!!formData.error ? (formData.error === "invalidEmail" ? t('releaseSubscription.error.invalidEmail') : t('releaseSubscription.error.networkError')) : ' '}
              />

              <LoadingButton
                type="submit"
                variant="contained"
                color="primary"
                loading={formData.loading}
                loadingIndicator={t('releaseSubscription.button.subscribing')}
                ml="1rem"
              >
                <Trans i18nKey="releaseSubscription.button.subscribe" />
              </LoadingButton>
            </Box>
```